sklearn/impute.py

```diff
@@ -917,8 +947,21 @@ def fit_transform(self, X, y=None):
             if self.verbose > 0:
                 print('[IterativeImputer] Ending imputation round '
                       '%d/%d, elapsed time %0.2f'
-                      % (i_rnd + 1, self.n_iter, time() - start_t))
-
+                      % (self.n_iter_, self.max_iter, time() - start_t))
+
+            if not self.sample_posterior:
+                inf_norm = np.linalg.norm(Xt - Xt_previous, ord=np.inf,
+                                          axis=None)
+                if inf_norm < normalized_tol:
+                    if self.verbose > 0:
+                        print('[IterativeImputer] Early stopping criterion '
```

Comment:
We also do not test the testing. Turning to `1` could be enough for a couple of tests. We will ensure that printing does not trigger an error.